cpp/src/generated/Tensor_generated.h

```diff
@@ -59,6 +59,7 @@ struct TensorDimBuilder {
         : fbb_(_fbb) {
     start_ = fbb_.StartTable();
   }
+  TensorDimBuilder &operator=(const TensorDimBuilder &);
```

Comment:
These declarations don't seem to cause any problem, but they are weird nevertheless, as the corresponding definitions do not exist. Which version of Flatbuffers did you use?

Using flatc version 23.5.26 (the latest release), I don't get these declarations.

Reply:
I used version `1.12.0` which is the version that generates the smallest diff and causes the least trouble.